games-puzzle/monsterz/monsterz-0.7.1.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A little arcade puzzle game"
+DESCRIPTION="Monsterz is a little arcade puzzle game, similar to the famous \
+Bejeweled or Zookeepe. The goal of the game is to create rows of similar \
+monsters, either horizontally or vertically. The only allowed move is the swap \
+of two adjacent monsters, on the condition that it creates a row of three or \
+more. When alignments are cleared, pieces fall from the top of the screen to \
+fill the board again. Chain reactions earn you even more points."
+HOMEPAGE="http://sam.zoy.org/monsterz/"
+COPYRIGHT="2007 Sam Hocevar, Brendan Rackley, MenTaLguY, Sun Microsystems\
+Michael Speck, David White Mike Kershaw"
+LICENSE="WTFPL v2
+	GNU GPL v1
+	GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="http://sam.zoy.org/monsterz/monsterz-$portVersion.tar.gz"
+CHECKSUM_SHA256="50828b8fa26d107bcc2bd134328f83c905b9f5e124846bdf239daf0eed34973d"
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	monsterz$secondaryArchSuffix = $portVersion
+	app:Monsterz$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+        "
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+	
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/Monsterz
```

Comment:
OK, all the files are now in their correct directories. That's fine.

Now, the hardest part: fixing INSTALL.
Because it currently does not install anything. It just creates an empty directory and a symlink to an inexisting file.
But I'm quite sure you will also need to pass some arguments to make in BUILD, like `prefix=$prefix`.

BTW, if you wish to create a Monsterz directory in $appsDir, you can use this variant:
```diff
-	mkdir -p $appsDir/Monsterz
+	install -d $appsDir/Monsterz
```